slu/slu/src/controller/prediction.py

```diff
@@ -69,6 +69,13 @@ def predict(
         """
         utterance = normalize(utterance)
 
+        if all(uttr in const.TEXTS_TO_IGNORE for uttr in utterances):
```

Comment:
Inclined to have this as a function in utils.py so that the behaviour is same in prepare and here.